swagger.yaml

```diff
@@ -183,8 +185,12 @@ paths:
           type: string
         - name: source_metadata
           in: query
-          description: '???'
+          description: >-
+            A JSON blob containing extra source specific information.  This
+            field is free-form, but we generally expect the data will follow
+            one of the formats outlined in the `/schema` folder.
```

Comment:
Oh, hrm, in production, you have to have the asset hash on the end (e.g. `https://api.monitoring.envirodatagov.org/assets/swagger-9c4ff43b64cc07ca722a29004093fd3ee86cdf46152aa7eab2f6fd3794f34736.yml`).

Not sure how we’d get that in a static YML file. I guess we could have a URL that serves up those files, but that is starting to feel a little excessive :\